forest/src/cli/mod.rs

```diff
@@ -339,12 +342,13 @@ pub(super) fn format_vec_pretty(vec: Vec<String>) -> String {
 
 /// convert `BigInt` to size string using byte size units (i.e. KiB, GiB, PiB, etc)
 /// Provided number cannot be negative, otherwise the function will panic.
-pub(super) fn to_size_string(input: &BigInt) -> String {
-    Byte::from_bytes(
-        u128::try_from(input).unwrap_or_else(|e| panic!("error parsing the input {input}: {e}")),
-    )
-    .get_appropriate_unit(true)
-    .to_string()
+pub(super) fn to_size_string(input: &BigInt) -> Result<String, String> {
```

Comment:
Perhaps let's use `anyhow::Result`?

Reply:
For now we are using basic `Result` everywhere there, so I use it too

Reply:
`anyhow::Result` potentially could help us with those nasty map_err statements. We use regularly all over the codebase but this could be a separate issue. @lemmih thoughts?